scripts/deploy-to-bluemix/bx_login.sh

```diff
@@ -1,14 +1,14 @@
 #!/bin/sh
```

Comment:
Is this script still needed, or is it superseded by scripts/bx_auth.sh?

Reply:
the scripts in the deploy-to-bluemix are for the toolchain environment. They are slightly different. Forlogging in bluemix, either the user enters a user/password or an API key. The bluemix installation also differs as it can't do `sudo ./Bluemix_CLI/install_bluemix_cli` (it returns an error). I think it's better if we have a separate script for a different environment